client/dashboard/app/router.tsx

```diff
@@ -139,6 +140,8 @@ const siteOverviewRoute = createRoute( {
 					queryClient.ensureQueryData( siteScanQuery( site.ID ) ),
 				hasHostingFeature( site, DotcomFeatures.BACKUPS ) &&
 					queryClient.ensureQueryData( siteLastBackupQuery( site.ID ) ),
+				hasPlanFeature( site, DotcomFeatures.SITE_PREVIEW_LINKS ) &&
+					queryClient.ensureQueryData( sitePreviewLinksQuery( site.ID ) ),
```

Comment:
It seems we wait for more and more endpoints here 🫠

Reply:
I updated this to is_a4a_dev_site since that's when we need to preload data.